itests/org.openhab.core.auth.oauth2client.tests/itest.bndrun

```diff
@@ -68,4 +68,5 @@ Fragment-Host: org.openhab.core.auth.oauth2client
 	net.bytebuddy.byte-buddy;version='[1.12.19,1.12.20)',\
 	net.bytebuddy.byte-buddy-agent;version='[1.12.19,1.12.20)',\
 	org.mockito.mockito-core;version='[4.11.0,4.11.1)',\
-	org.objenesis;version='[3.3.0,3.3.1)'
+	org.objenesis;version='[3.3.0,3.3.1)',\
+	org.osgi.service.cm;version='[1.6.0,1.6.1)'
```

Comment:
Are all these changes of adding `org.osgi.service.cm` intended? It does not seem to be related to this PR, so it's probably better to revert them.